sdk/core/azure-core/azure/core/polling/base_polling.py

```diff
@@ -430,29 +588,35 @@ def initialize(self, client, initial_response, deserialization_callback):
 
         except BadStatus as err:
             self._status = "Failed"
-            raise HttpResponseError(response=initial_response.http_response, error=err)
+            raise HttpResponseError(response=initial_response.http_response, error=err) from err
         except BadResponse as err:
             self._status = "Failed"
-            raise HttpResponseError(response=initial_response.http_response, message=str(err), error=err)
+            raise HttpResponseError(response=initial_response.http_response, message=str(err), error=err) from err
         except OperationFailed as err:
-            raise HttpResponseError(response=initial_response.http_response, error=err)
+            raise HttpResponseError(response=initial_response.http_response, error=err) from err
 
     def get_continuation_token(self) -> str:
         import pickle
 
         return base64.b64encode(pickle.dumps(self._initial_response)).decode("ascii")
 
     @classmethod
-    def from_continuation_token(cls, continuation_token: str, **kwargs) -> Tuple[Any, Any, Callable]:
+    def from_continuation_token(
+        cls, continuation_token: str, **kwargs
+    ) -> Tuple[Any, Any, Callable[[Any], PollingReturnType_co]]:
         try:
             client = kwargs["client"]
         except KeyError:
-            raise ValueError("Need kwarg 'client' to be recreated from continuation_token")
+            raise ValueError(  # pylint: disable=raise-missing-from
```

Comment:
As above, we could use `from None` here.

Reply:
Yep